examples/build.gradle.kts

```diff
@@ -5,10 +5,10 @@ import org.jetbrains.kotlin.gradle.targets.js.webpack.KotlinWebpackConfig
 
 plugins {
   alias(libs.plugins.compose.compiler)
-  alias(libs.plugins.storytale)
   alias(libs.plugins.kotlinMultiplatform)
   alias(libs.plugins.androidApplication)
   alias(libs.plugins.jetbrainsCompose)
+  id("org.jetbrains.compose.storytale")
```

Comment:
this change was made because we want to set the version via an argument for a gradle task, not via libs.versions.toml

Reply:
JIC: it's possible to override version coming from version catalog ([example](https://github.com/whyoleg/cryptography-kotlin/blob/69b77a88b4b81109704475ed02be48d263d1a1c8/build-settings/src/main/kotlin/cksettings.kotlin-version-override.settings.gradle.kts#L36))

So you can probably just add to `settings.gradle.kts`:
```
val deployVersion = providers.gradleProperty("storytale.deploy.version").orNull?.takeIf(String::isNotBlank)

if (deployVersion != null) {
    dependencyResolutionManagement {
        versionCatalogs.register("libs") {
            version("storytale", deployVersion)
        }
    }
}
```